### PR TITLE
renamed pebble to fusion-studio

### PR DESCRIPTION
**Pull request by ccheraa**

### CODE REVIEW

File: browser-app/package.json [hunk 1]
```diff
@@ -3,24 +3,24 @@
...
+    "@theia/process": "latest",
+    "@theia/terminal": "latest",
+    "@theia/editor": "latest",
+    "@theia/languages": "latest",
+    "@theia/markers": "latest",
+    "@theia/monaco": "latest",
+    "@theia/typescript": "latest",
+    "@theia/messages": "latest",
+    "@theia/textmate-grammars": "latest",
+    "fusion-studio-extension": "0.1.0"
```
Comment (adamretter) [inline browser-app/package.json]: Should this not be `0.6.0` as that is what you set in `fusion-studio-extension/package.json`

File: electron-app/package.json [hunk 2]
```diff
@@ -3,24 +3,25 @@
...
+    "@theia/terminal": "latest",
+    "@theia/editor": "latest",
+    "@theia/languages": "latest",
+    "@theia/markers": "latest",
+    "@theia/monaco": "latest",
+    "@theia/typescript": "latest",
+    "@theia/messages": "latest",
+    "@theia/electron": "latest",
+    "@theia/textmate-grammars": "latest",
+    "fusion-studio-extension": "0.1.0"
```
Comment (adamretter) [inline electron-app/package.json]: Should this not be `0.6.0` as that is what you set in `fusion-studio-extension/package.json`

**Comment by ccheraa** [comment]
You're right, I should change it to "latest", I'll do it.

**Comment by adamretter** [comment]
Sorry @ccheraa I saw an older PR from you too for improving permissions, so I merged that first. Could you take a look at the merge conflicts on this one now? Sorry.

**Comment by adamretter** [comment]
@ccheraa I resolved the conflict and merged this manually. Thank you for your changes :-)

**Pull request closed without merging**